authors/apps/authentication/tests/test_social_auth.py

```diff
@@ -0,0 +1,59 @@
+import os
+from django.urls import reverse
+from rest_framework.test import APITestCase, APIClient
+from rest_framework import status
+
+AUTH_URL = reverse('authentication:social')
+
+
+class SocialAuthTest(APITestCase):
+    """
+    Base tet class for social authentication
+    """
+    def setUp(self):
+        self.client = APIClient()
+
+    def authenticate(self, provider="google-oauth2",
+                     access_token_secret=os.environ.get('GOOGLE_ACCESS_TOKEN_SECRET'),
```

Comment:
You may need to add default values so the tests don't experience errors due to issues with the environment